src/lib.rs

```diff
@@ -155,10 +146,6 @@ struct CommonOptions {
     #[structopt(long, parse(from_os_str), value_name = "CONFIG_PATH")]
     config: Option<PathBuf>,
 
-    /// Use Cranelift for all compilation
-    #[structopt(long, conflicts_with = "lightbeam")]
-    cranelift: bool,
```

Comment:
Maybe keep this for in the future if a new backend is added?

Reply:
This can be added in the future and otherwise I think it would be confusing to have a flag that basically does nothing.